gradio/components/multimodal_textbox.py

```diff
@@ -0,0 +1,196 @@
+"""gr.MultimodalTextbox() component."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Literal
+
+import gradio_client.utils as client_utils
+from gradio_client.documentation import document
+
+from gradio import processing_utils
+from gradio.components.base import FormComponent
+from gradio.data_classes import FileData, GradioRootModel
+from gradio.events import Events
+
+
+class MultimodalData(GradioRootModel):
+    root: Dict[str, str | List[FileData]]
+
+
+@document()
+class MultimodalTextbox(FormComponent):
+    """
+    Creates a textarea for users to enter string input or display string output and also allows for the uploading of multimedia files.
+
+    Demos: chatbot_multimodal
+    Guides: creating-a-chatbot
+    """
+
```

Comment:
We should set `data_model = MultimodalData` here so that we can get rid of the `api_info` method.

Right now the return value is incorrect and the View API page does not show the right data
<img width="710" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/511f7c0c-f29c-4ee9-a45b-b10a03f1519a">